src/App.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react';
+import Page from './Page';
+
+function App() {
+  const [state, setState] = useState({
+    count: 0,
+  });
+  const { count } = state;
+  function buttonClick(value = 1) {
+    setState(
+      {
+        count: count + value
+        ,
+      },
+    );
+  }
+  return (
+    <Page
+      count={count}
+      onClick={buttonClick}
+    />
+  );
+}
```

Comment:
```suggestion
function App() {
  const [state, setState] = useState({
    count: 0,
  });
  const { count } = state;
  
  function buttonClick(value = 1) {
    setState({
        count: count + value,
    });
  }
  
  return (
    <Page
      count={count}
      onClick={buttonClick}
    />
  );
}
```

우리가 글쓰기를 할 때 내용상으로 구분이 되는 경우 문단을 넣어주듯이 코드도 마찬가지에요.

의미상으로 구분을 주기 위해 공백을 사용하면 코드가독성을 높이는데 도움이 돼요.

공백을 넣었을 때랑 비교하면 어떠하신가요?

Reply:
내용상 구분되는 코드를 쉽게 알아볼 수 있게 된것 같네요